tests-js/loan-settings/LoanSettings.js

```diff
@@ -0,0 +1,156 @@
+
+		await testWrbtc.mint(sovryn.address, ether("500"));
+
+        loanParams = {
+            "id": "0x0000000000000000000000000000000000000000000000000000000000000000",
+            "active": false,
+            "owner": constants.ZERO_ADDRESS,
+            "loanToken": underlyingToken.address,
+            "collateralToken": testWrbtc.address,
+            "minInitialMargin": ether("50"),
+            "maintenanceMargin": ether("15"),
+            "maxLoanTerm": "2419200"
+        };
+   
+        let tx = await sovryn.setupLoanParams([Object.values(loanParams)]);
+        loanParamsId = tx.logs[1].args.id;
+	});
+
+    
+	describe("test LoanSettings", async () => {
+		it("test setup removeLoanParams", async () => {
+            let loanParamsAfter = (await sovryn.getLoanParams([loanParamsId]))[0];
+            
+            assert(loanParamsAfter["id"] != "0x0");
+            assert(loanParamsAfter["active"]);
+            assert(loanParamsAfter["owner"] == lender);
+            assert(loanParamsAfter["loanToken"] == underlyingToken.address);
+
+
+            await expectRevert(sovryn.disableLoanParams([loanParamsId], { "from": accounts[0] }), "unauthorized owner");
+    
+            await sovryn.disableLoanParams([loanParamsId], { "from": lender });
+            assert((await sovryn.getLoanParams([loanParamsId]))[0]["id"] != "0x0");
+		});
+
+ 
+        it("test disableLoanParams", async () => {
+            await sovryn.disableLoanParams([loanParamsId], { "from": lender });
+
+            let loanParamsAfter = (await sovryn.getLoanParams([loanParamsId]))[0];
+
+            assert(loanParamsAfter["id"] != "0x0");
+            assert(loanParamsAfter["active"] == false); // false because we disabled Loan Param just before
+            assert(loanParamsAfter["owner"] == lender);
+            assert(loanParamsAfter["loanToken"] == underlyingToken.address);
+            assert(loanParamsAfter["collateralToken"] == testWrbtc.address);
+            assert(loanParamsAfter["minInitialMargin"] == ether("50"));
+            assert(loanParamsAfter["maintenanceMargin"] == ether("15"));
+            assert(loanParamsAfter["maxLoanTerm"] == "2419200");
+		});
+
+
+        it("test getLoanParams", async () => {
+            let loanParamsAfter = (await sovryn.getLoanParams([loanParamsId]))[0];
+
+            assert(loanParamsAfter["id"] != "0x0");
+            assert(loanParamsAfter["active"]); 
+            assert(loanParamsAfter["owner"] == lender);
+            assert(loanParamsAfter["loanToken"] == underlyingToken.address);
+            assert(loanParamsAfter["collateralToken"] == testWrbtc.address);
+            assert(loanParamsAfter["minInitialMargin"] == ether("50"));
+            assert(loanParamsAfter["maintenanceMargin"] == ether("15"));
+            assert(loanParamsAfter["maxLoanTerm"] == "2419200");
+		});
+
+
+		it("test getLoanParamsList", async () => {
```

Comment:
pls consider using prettier that is set as the uniform formatter to avoid misleading format differences further on